shared/modules/browser-runtime.utils.js

```diff
@@ -53,3 +58,13 @@ export function checkForLastErrorAndWarn() {
 
   return error;
 }
+
+export function getIsBrowserPrerenderBroken(
```

Comment:
Nit: A JSDoc comment would be appreciated here!

Reply:
good idea https://github.com/MetaMask/metamask-extension/pull/24317/commits/821221265383002f1de0cde8765d154f2a9047a6